packages/earth-map/src/components/searchbox/component.tsx

```diff
@@ -17,17 +17,21 @@
   specific language governing permissions and limitations under the License.
 */
 
-import React, { useEffect, useState } from 'react';
+import React from 'react';
 import classnames from 'classnames';
 
-import Link from 'redux-first-router-link';
+import { noop } from 'utils';
```

Comment:
TS2305: Module '"../../utils"' has no exported member 'noop'.

Reply:
Try opening the utils file. TS is sometimes confused. The export is there